pyfesom2/fesom2GeoFormat.py

```diff
@@ -0,0 +1,90 @@
+from pyfesom2 import fesom2regular
+from osgeo import osr, gdal
+import numpy as np
+
+
+def fesom2GeoFormat(data, mesh, outName, radius_of_influence=100000, methode='nn', driver='Gtiff'):
+    """ Returns Fesom as Gdal Geotiff or Gdal NetCDF .
+
+    Parameters:
+        data (array): which is come from select_slices().
+        mesh (fesom mesh): which is come from load_mesh().
+        outName (str): output Name.
+        methode : interpolation methode for Fesom2Regular().
+        driver(str): driver name for Gdal ( Gtiff or NetCDF).
+
+    Returns:
+        Gdal Datasets:Gtiff or NetCDF.   
+
+    """
+    lonreg, latreg, rasterOrigin = grid(mesh)
+    data2 = dataShape(data)
+    pixelWidth = 1
+    pixelHeight = -1
+    outputName = outName
+    array = fesom2regular(
+        data2,
+        mesh,
+        lonreg,
+        latreg,
+        distances_path=None,
+        inds_path=None,
+        qhull_path=None,
+        how=methode,
```

Comment:
plaese rename

Reply:
of course :), I am going to change it, thanks for your review.